source/get-started/create-a-deployment.txt

```diff
@@ -0,0 +1,32 @@
+.. _cpp-quick-start-create-deployment:
+
+===========================
+Create a MongoDB Deployment
+===========================
+
+You can create a free tier MongoDB deployment on MongoDB Atlas
+to store and manage your data. MongoDB Atlas hosts and manages
+your MongoDB database in the cloud.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a Free MongoDB deployment on Atlas
+
+      Complete the :atlas:`Get Started with Atlas </getting-started?tck=docs_driver_cpp>`
```

Comment:
Q: is this url addition doing anything? `?tck=docs_driver_cpp`

Reply:
Followed up with Jordan about this, since he wrote the corresponding Pymongo page: seems like it's to track the traffic but isn't really being used. I'll take it out here